MAINTAINERS.md

```diff
@@ -0,0 +1,197 @@
+# Maintainers
+
+This document serves to clarify the responsibilities of a maintainer as well as
+serve as a wiki for maintainers to link to / include resources.
+
+
+## Current Maintainers
+
+The current maintainers are:
+
+| name | focus areas |
+------------------
+|   @achow101  | wallet, PSBTs |
+|   @fanquake | general, tooling, build system |
+|   @hebasto | UI/UX |
+|   @laanwj | general |
+|   @marcofalke | general |
+|   @sipa | general, peer-to-peer, consensus, cryptography |
```

Comment:
Should @sipa still be in this list? https://github.com/bitcoin/bitcoin/pull/25568 https://twitter.com/pwuille/status/1545159642447335424